src/Xamarin.Android.Build.Tasks/Tasks/CreateMultiDexMainDexClassList.cs

```diff
@@ -32,13 +35,21 @@ public override bool Execute ()
 			Log.LogDebugTaskItems ("  CustomMainDexListFiles:", CustomMainDexListFiles);
 			Log.LogDebugMessage ("  ToolExe: {0}", ToolExe);
 			Log.LogDebugMessage ("  ToolPath: {0}", ToolPath);
-			
+			Log.LogDebugMessage ("  ProguardHome: {0}", ProguardHome);
+
 			if (CustomMainDexListFiles != null && CustomMainDexListFiles.Any ()) {
 				var content = string.Concat (CustomMainDexListFiles.Select (i => File.ReadAllText (i.ItemSpec)));
 				File.WriteAllText (MultiDexMainDexListFile, content);
 				return true;
 			}
 
+			// Windows seems to need special care.
+			// On the other hand, xbuild has a bug and fails to parse '=' in the value, so we skip JAVA_TOOL_OPTIONS on Mono runtime.
+			EnvironmentVariables =
+				OS.IsWindows ?
```

Comment:
@radical: Is there some way to have an "xbuild" check instead of an OS check here?

The problem is the following line: `JAVA_TOOL_OPTIONS=-Dfile.encoding=UTF8`. xbuild doesn't properly split on `=`, causing the second `=` to be "lost".

Ideally we'd specify `JAVA_TOOL_OPTIONS` whenever we're using MSBuild -- as MSBuild properly splits on `=` -- and omit `JAVA_TOOL_OPTIONS` when using xbuild.